va_explorer/templates/va_data_management/show.html

```diff
@@ -31,9 +31,31 @@
 <div class="tab-content">
   <div class="tab-pane fade show active" id="record" role="tabpanel" aria-labelledby="record-tab">
     <div class="row mt-3">
-      {% if perms.va_data_management.change_verbalautopsy %}
-        <a class="btn btn-primary" href="{% url 'data_management:edit' id=id %}">Edit Record</a>
-      {% endif %}
+        <div class="va-options-container">
+            <div class="checkbox-container" align='center'>
+              <input type="checkbox" id="hidden-missing" name="hidden-missing" value="checked" checked="TRUE" style="margin-left: 20px;">
+              <label data-toggle="tooltip" 
+                    title="Toggle whether or not you want to see fields with 'N/A' or '' as responses. When unchecked, these blank responses will be shown." 
+                    for="hidden-missing" 
+                    style="margin-left: 5px; font-size:18px;">
```

Comment:
Since both labels have the same inline style, suggest extracting out into a class (like you did for va-options-container)

`style="margin-left: 5px; font-size:18px;"`